### DISCUSSION TITLE
Behind Proxy

### DISCUSSION THREAD

**Discussion by ariefsn** [urql-graphql/urql#3646] [Q&A] [answered]
Hi,
I have a question.
How to use `urql` with a proxy?
So my backend is behind caddy proxy.
I'm not sure why it failed if I used the proxied URL.

![image](https://github.com/user-attachments/assets/68272bf0-fbf8-4c28-ae47-2d9276c2e436)

It's working fine if I use the original URL without proxy. 

And if I use the GraphQL playground directly within the proxied URL, it works well too 😔

Thanks,
Arief

**Comment by kitten** [comment, marked as answer]
Check you network tab. If I had to guess, your Browser probably doesn't trust the self-signed certificate on that, or you have your local app running on a different testing domain and CORS is failing

**Reply by ariefsn** [reply]
hi @kitten 
You're right, sorry 🙏🏻.
The caddy proxy was not configured properly.

Thanks

**Reply by kitten** [reply]
Nice! Glad you sorted it out 🫶
